src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -821,6 +970,12 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         return result;
     }
     
+    /**
+     * Returns a list of courseIds for all archived courses for all instructors.
```

Comment:
Either way, do also change the first word of all the descriptions to either `Gets` or `Returns` where applicable for consistency :)